environment.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
   - forcefield-utilities=0.2.1
   - foyer=0.11.3
   - gmso=0.9.1
-  - mbuild=0.15.1
+  - mbuild
```

Comment:
Let us keep these set and push the changed versions as they come unless there is a reason not to... ?

so set as

mbuild=0.16.0